3ds Max/Max2Babylon/Exporter/BabylonExporter.GLTFExporter.Texture.cs

```diff
@@ -174,60 +175,60 @@ private GLTFTextureInfo ExportTexture(BabylonTexture babylonTexture, GLTF gltf,
-                var gltfTexture = new GLTFTexture
-                {
-                    name = name,
-                    sampler = gltfSampler.index,
-                    source = gltfImage.index
-                };
-                gltfTexture.index = gltf.TexturesList.Count;
-                gltf.TexturesList.Add(gltfTexture);
-
-
-                // --------------------------
-                // ------ TextureInfo -------
-                // --------------------------
-                var gltfTextureInfo = new GLTFTextureInfo
-                {
-                    index = gltfTexture.index,
-                    texCoord = babylonTexture.coordinatesIndex
-                };
+            // --------------------------
+            // -------- Texture ---------
+            // --------------------------
+
+            RaiseMessage("GLTFExporter.Texture | create texture", 3);
+            var gltfTexture = new GLTFTexture
+            {
+                name = name,
+                sampler = gltfSampler.index,
+                source = gltfImage.index
+            };
+            gltfTexture.index = gltf.TexturesList.Count;
+            gltf.TexturesList.Add(gltfTexture);
+
+
+            // --------------------------
+            // ------ TextureInfo -------
+            // --------------------------
+            var gltfTextureInfo = new GLTFTextureInfo
+            {
+                index = gltfTexture.index,
+                texCoord = babylonTexture.coordinatesIndex
+            };
 
                 if (!(babylonTexture.uOffset == 0) || !(babylonTexture.vOffset == 0) || !(babylonTexture.uScale == 1) || !(babylonTexture.vScale == -1) || !(babylonTexture.wAng == 0))
+            {
+                // Add texture extension if enabled in the export settings
+                if (exportParameters.enableKHRTextureTransform)
+                {
+                    AddTextureTransformExtension(ref gltf, ref gltfTextureInfo, babylonTexture);
+                }
+                else
                 {
-                    // Add texture extension if enabled in the export settings
-                    if (exportParameters.enableKHRTextureTransform)
-                    {
-                        AddTextureTransformExtension(ref gltf, ref gltfTextureInfo, babylonTexture);
-                    }
-                    else
-                    {
                         RaiseWarning("GLTFExporter.Texture | KHR_texture_transform is not enabled, so the texture may look incorrect at runtime!", 3);
-                    }
                 }
+            }
                 var textureID = name + TextureTransformID(gltfTextureInfo);
                 // Check for texture optimization.  This is done here after the texture transform has been potentially applied to the texture extension
                 if (CheckIfImageIsRegistered(textureID))
                 {
                     var textureComponent = GetRegisteredTexture(textureID);
-
+            
```

Comment:
whitespace changes here as well, please use a separate PR to correct whitespace where there's no logic change.